.github/workflows/test.yml

```diff
@@ -7,7 +7,8 @@ on:
   pull_request:
 
 env:
-  DATABASE_URL: "mysql://goteo:goteo@127.0.0.1:3307/goteo?serverVersion=10.2"
+  GOTEO_DATABASE_URL: "mysql://goteo:goteo@127.0.0.1:3307/goteo?serverVersion=10.2"
+  OAUTH_DATABASE_URL: "mysql://goteo:goteo@127.0.0.1:3307/goteo?serverVersion=10.2"
```

Comment:
There's no possibility to connect with the Goteo DB from the GitHub test environment. I've added these lines only to prevent tests from failing. However, they should be pointing to `mariadb_oauth` instead of `127.0.0.1`. I'm going to change it.

Reply:
I've just updated both connections with more appropriate ones.